eventhubs/client/azure-eventhubs/src/test/java/com/azure/eventhubs/EventHubClientBuilderTest.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import org.junit.Test;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Locale;
+
+public class EventHubClientBuilderTest {
+    private static final String END_POINT_FORMAT = "sb://%s.%s";
+    private static final String NAMESPACE_NAME = "dummyNamespaceName";
+    private static final String DEFAULT_DOMAIN_NAME = "servicebus.windows.net/";
+    private static final String SHARED_ACCESS_KEY_NAME = "dummySasKeyName";
+    private static final String SHARED_ACCESS_KEY = "dummySasKey";
+    private static final String END_POINT_STR = getURI(END_POINT_FORMAT, NAMESPACE_NAME, DEFAULT_DOMAIN_NAME).toString();
+
+    private static final String PROXY_HOST = "/127.0.0.1";
+    private static final String PROXY_PORT = "3128";
+
+    private static final String correctConnStr = String.format("Endpoint=%s;SharedAccessKeyName=%s;SharedAccessKey=%s",
+        END_POINT_STR, SHARED_ACCESS_KEY_NAME, SHARED_ACCESS_KEY);
+    private static Proxy proxyAddress = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(PROXY_HOST, Integer.parseInt(PROXY_PORT)));
+
+    private static final CredentialInfo credentialInfo = new CredentialInfo(correctConnStr);
+
+    private static URI getURI(String endpointFormat, String namespaceName, String domainName) {
```

Comment:
In general, move your public methods to the top. Then private methods to the bottom. Users will interact with public methods first before looking at these private helper methods.

Reply:
Private method is still up here.